coq/Makefile

```diff
@@ -0,0 +1,870 @@
+##########################################################################
+##         #   The Coq Proof Assistant / The Coq Development Team       ##
+##  v      #         Copyright INRIA, CNRS and contributors             ##
+## <O___,, # (see version control and CREDITS file for authors & dates) ##
+##   \VV/  ###############################################################
+##    //   #    This file is distributed under the terms of the         ##
```

Comment:
Isn't this file duplicated with `coq/CoqMakefile` ?

Reply:
Yep, deleting it.